CRM/Utils/Date.php

```diff
@@ -2122,32 +2131,28 @@ public static function formatDate($date, $dateType) {
 
     if (CRM_Utils_Date::convertToDefaultDate($dateParams, $dateType, $dateKey)) {
       $dateVal = $dateParams[$dateKey];
-      $ruleName = 'date';
-      if ($dateType == 1) {
+      if ($dateType === 1) {
```

Comment:
This breaks it because $dateType is string '1'.

Reply:
argh - I got ahead of myself

Reply:
ok - now passing an int but not requiring one